x-pack/plugins/fleet/server/services/epm/elasticsearch/template/template.test.ts

```diff
@@ -23,27 +29,29 @@ expect.addSnapshotSerializer({
 });
 
 test('get template', () => {
-  const templateName = 'logs-nginx-access-abcd';
+  const templateIndexPattern = 'logs-nginx-access-abcd-*';
```

Comment:
Nit: I know it is a test but I would try to align the tests naming vice with the indexing strategy so this would be `log-nginx.access-...`

Reply:
Done.